### DISCUSSION TITLE
Greasyfork 上存在一份疑似违反 LICENCE 的副本，并且疑似盗搬

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4046] [综合讨论 / General]
链接: https://greasyfork.org/zh-CN/scripts/460764

用户: https://greasyfork.org/zh-CN/users/1033266

[这是 Greasyfork 上副本的备份(COPY.txt)](https://github.com/the1812/Bilibili-Evolved/files/10891324/COPY.txt)

根据 [LICENCE](https://github.com/the1812/Bilibili-Evolved/blob/master/LICENCE.md) 中对“限制”的相关说明，个人认为该用户的行为违反了本仓库的 LICENCE

是否遵守了“限制”第一条？看起来并没有:
不仅未保留 README 文档，而且安装入口并非 README 文档中指定的链接
![图片](https://user-images.githubusercontent.com/81268244/222949145-a84e300c-cb6a-4587-8016-65ddaabf80d1.png)

是否遵守了“限制”第二条？看起来并没有:
跳转到 COPY.txt 第105行的第41170字符，可以发现其中一个指向本仓库的反馈渠道并没有被替换或移除

**Comment by the1812** [comment]
是盗搬

(顺便还发现我 LICENCE 里的链接写错了)

**Comment by Tinhone** [comment]
感觉我还摸不透 Greasyfork 的举报系统，也不知道需要提供什么信息才能顺利让脚本删除...等一位大佬orz

**Reply by the1812** [reply]
我点举报了 选未经授权的副本加上原地址就行

**Reply by Tinhone** [reply]
原来如此，提交举报了

**Comment by Geeyun-JY3** [comment]
不是第一次，估计也不会是最后一次：#3277
